app/models/platform.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class Platform < ApplicationRecord
+	def self.platform_by_host(host)
+		return if Platform.count == 0
```

Comment:
here we would default to one

Reply:
I created platfrom_id as an optional value for user table
So, if there is no platform in db we will assign nil by default
Cause MySQL cant link to platform table with non-existent ID